demos/qadi_experiments/qadi_google_real.py

```
@@ -0,0 +1,141 @@
+# Load .env
+env_path = Path(__file__).parent / '.env'
+if env_path.exists():
+    with open(env_path) as f:
+        for line in f:
+            if line.strip() and '=' in line and not line.startswith('#'):
+                key, value = line.strip().split('=', 1)
+                os.environ[key] = value.strip('"').strip("'")
+
+sys.path.insert(0, str(Path(__file__).parent / 'src'))
+
+async def run_minimal_qadi(prompt: str):
```

Ruff (0.11.9)

`run_minimal_qadi` is too complex (12 > 10)

(C901)